src/v1/base/Menu.tsx

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+} from 'react';
+import { IconHamburger } from '@public/icons';
+import BottomSheet from './BottomSheet';
+
+type MenuContextValue = {
+  open: boolean;
+  toggle: () => void;
+};
+
+const MenuContext = createContext({} as MenuContextValue);
+
+const Menu = ({ children }: { children?: React.ReactNode }) => {
+  const [open, setOpen] = useState(false);
```

Comment:
```suggestion
  const [isOpened, setOpen] = useState(false);
```

is접미사를 통해 해당 변수 타입이 boolean임을 암묵적으로 알려줄 수도 있을것 같스비다!